test/03-device-state-v2.ts

```diff
@@ -333,6 +333,64 @@ describe('Device State v2', () => {
 			);
 		});
 	});
+
+	describe(`Tracking latest release`, () => {
```

Comment:
@Page- i'm not sure if these tests belong in this state endpoint file, or elsewhere as a "scenario".

Reply:
I'm happy for these to be in tests but I'd put them as separate device pinning tests since that's what they're mainly testing rather than the device state per se (they're checking the device state response as a way to verify the pinning but that could just as easily be done via an OData GET)